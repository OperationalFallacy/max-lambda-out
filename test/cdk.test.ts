import { SynthUtils } from '@aws-cdk/assert';
import { Stack } from '@aws-cdk/core';
import * as s from '../src/api-stack';

test('dlq creates an alarm', () => {
  const stack = new Stack();
  new s.ApiStack(stack, 'Stack');
  expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
});