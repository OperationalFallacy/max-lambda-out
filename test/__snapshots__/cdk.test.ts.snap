// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dlq creates an alarm 1`] = `Object {}`;
